images/fcos-base-image/root/usr/lib/systemd/system/bind9.service

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=Creates the bind9 name server service
+Wants=network-online.target
+After=network-online.target
+RequiresMountsFor=%t/containers
+
+[Service]
+Environment=PODMAN_SYSTEMD_UNIT=%n
+Restart=on-failure
+TimeoutStopSec=70
+ExecStartPre=/bin/rm -f %t/%n.ctr-id
+ExecStart=/usr/bin/podman run --rm --log-driver=journald -d -p 53:53 --name bind9 -v /opt/bind9_zones:/var/lib/bind -v /opt/bind9/named.conf:/etc/named.conf registry.ci.openshift.org/ci/bmqe-bind9:latest 
```

Comment:
@jadhaj this unit should be defined in the fcos-bastion-image folder (defines the new root folder at need).

Also, add the /opt/bind9_zones folder and /opt/bind9/named.conf files to git (unless they expose sensitive information).

Reply:
> Not sure where to add it. I see that all service are under:
images/fcos-base-image/root/usr/lib/systemd/system/

Create the same folders structure in images/fcos-bastion-images. For example, https://github.com/openshift-qe/baremetal-qe-infra/pull/14/files does it already.

> As for opt/bind9_zones folder and /opt/bind9/named.conf file: where to place them ?

Create a folder `root/usr/share/bastion_services/bind9` and place the `named.conf` file there.

Ignore /opt/bind9_zones for now instead, but please add it in the PR description for me to remember of it.

Something like:

> This code expects the /opt/bind9_zones folder with the files x and y to be provided via ignition.